fibonacci/fibonacci.js

```diff
@@ -4,9 +4,7 @@ const fibonacci = function(count) {
   let a = 0;
   let b = 1;
   for (let i = 1; i < count; i++) {
-    const temp = b;
-    b = a + b;
-    a = temp;
+    [a, b] = [b, a+b];
```

Comment:
Considering the level most users might be at, I'm not sure if this would be an ideal solution to present. Other than being slightly shorter (which I don't think should be the purpose of these exercises), is there another benefit to this?

Reply:
I've started theodinproject from zero knowledge about programming and I find that solution on my own. When I compared with the given solution, I found that the intermediate value confused my mind. So I would like to share mine.  I may not have the necessary hindsight to know if this is relevant but it can be given as an additional solution to the exercise. It's up to you. ;-) However thanks to answer my PR ;-)

Reply:
@Florent-V Maybe we could add your proposal here as a comment in the file, something like:

```suggestion
    // A slightly shorter version of the above solution:
    // [a, b] = [b, a + b];
```

Reply:
Yes it's a good idea !